Src/Couchbase.Linq.IntegrationTests/App.config

```diff
@@ -39,7 +39,7 @@
       </dependentAssembly>
       <dependentAssembly>
         <assemblyIdentity name="Castle.Core" publicKeyToken="407dd0808d44fbdc" culture="neutral" />
-        <bindingRedirect oldVersion="0.0.0.0-4.1.1.0" newVersion="4.1.1.0" />
+        <bindingRedirect oldVersion="0.0.0.0-4.2.1.0" newVersion="4.0.0.0" />
```

Comment:
What is the motivation for this change?

Reply:
@jeffrymorris 

As near as I can tell, they redid the DLL versioning between versions of Castle.Core and started versioning all of them as 4.0.0.0 for strong naming consistency.  Doesn't work without this binding redirect.